Kitodo/src/main/resources/messages/messages_en.properties

```diff
@@ -532,21 +532,21 @@ granularity.info.legend=Your choice\:
 granularity.info.noNumberOfPages={0} Kitodo processes will be laid out.
 granularity.info.textbox.batches=In order to optimise the logistics of the excavation, transport and processing, the generated processes are automatically combined into batches per {0}.
 granularity.info.textbox.caption=Explanation
-granularity.info.textbox.days=Each process represents a single calendar day from the course of appearance of the newspaper, containing all issues actually published that day. Processes are laid out only for days where an issues actually has published. With only one issue per day, this selection is identical with the option \u201Cissues\u201D.
+granularity.info.textbox.days=Each process represents a single calendar day from the course of appearance of the newspaper, containing all issues actually published that day. Processes are laid out only for days where an issues actually has published. With only one issue per day, this selection is identical with the option “issues”.
 granularity.info.textbox.glyph=i
 granularity.info.textbox.issues=Processes are laid out for actually published editions only. As long as the calendar day selection can link to exactly one process only, having several editions on the same date means that the other issues are not searchable in this way.
-granularity.info.textbox.months=Each process represents a single calendar month of the course of publication of the newspaper, which contains all this month actually published editions of the newspaper. Since the DFG Viewer\u2019s calendar day selection can currently (as of 2014) only link to the processes level, this has the consequence that it will always be jumping to the first image of the month, regardless of the day chosen, therefore\u2212in most cases\u2212on the wrong day.
+granularity.info.textbox.months=Each process represents a single calendar month of the course of publication of the newspaper, which contains all this month actually published editions of the newspaper. Since the DFG Viewer's calendar day selection can currently (as of 2014) only link to the processes level, this has the consequence that it will always be jumping to the first image of the month, regardless of the day chosen, therefore−in most cases−on the wrong day.
```

Comment:
```suggestion
granularity.info.textbox.months=Each process represents a single calendar month of the course of publication of the newspaper, which contains all this month actually published editions of the newspaper. Since the DFG Viewer's calendar day selection can currently (as of 2014) only link to the processes level, this has the consequence that it will always be jumping to the first image of the month, regardless of the day chosen, therefore - in most cases - on the wrong day.
```